Makefile

```diff
@@ -0,0 +1,41 @@
+PROJ=zetcd
+ORG_PATH=github.com/coreos
+REPO_PATH=$(ORG_PATH)/$(PROJ)
+export PATH := $(PWD)/bin:$(PATH)
+
+VERSION ?= $(shell ./scripts/git-version)
+
+DOCKER_IMAGE=coreos/zetcd:$(VERSION)
+
+$( shell mkdir -p bin )
+$( shell mkdir -p _output/images )
+$( shell mkdir -p _output/bin )
+
+user=$(shell id -u -n)
+group=$(shell id -g -n)
+
+export GOBIN=$(PWD)/bin
+export PATH=$(GOBIN):$(shell printenv PATH)
+export GO15VENDOREXPERIMENT=1
+
+LD_FLAGS="-w -X $(REPO_PATH)/version.Version=$(VERSION)"
+
+release-binary:
+	@go build -o $(GOPATH)/bin/zetcd -v -ldflags $(LD_FLAGS) $(REPO_PATH)/cmd/zetcd
+
+_output/bin/zetcd:
+	@./scripts/docker-build
+	@sudo chown $(user):$(group) _output/bin/zetcd
```

Comment:
I tried running `docker-build` and it seems the `docker cp` will copy into a file owned by the user invoking it, so is this chown needed?